components/base/CardGrid.tsx

```diff
@@ -26,16 +30,21 @@ const CardWrapper = styled.div`
     text-decoration: none;
   }
 `;
+
 type Props = {
   children: JSX.Element[];
   variant?: "default" | "center";
+  maxColumns?: 1 | 2 | 3 | 4;
```

Comment:
We can now specify a maximum number of columns to use. This was useful for the partnerships page because the "On this Page" reduced the available width for the other content, so the four column layout looked too tight, even on larger displays